tests/integration/test_logcollector/test_statistics/test_statistics_macos.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+import pytest
+import wazuh_testing.tools.configuration as conf
+
+from json import load
+from wazuh_testing import logcollector
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools import LOGCOLLECTOR_STATISTICS_FILE
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.tools.file import read_json
+
+# Marks
+pytestmark = [pytest.mark.darwin, pytest.mark.tier(level=1)]
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_statistics_macos.yaml')
+
+parameters = [
+    {'LOCATION': 'macos', 'LOG_FORMAT': 'macos'},
+]
+
+metadata = [
+    {'location': 'macos', 'log_format': 'macos'}
+]
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['LOCATION']}_{x['LOG_FORMAT']}" for x in parameters]
+
+local_internal_options = {'logcollector.state_interval': 1}
+
+
+# Fixtures
+@pytest.fixture(scope="module")
+def get_local_internal_options():
+    """Get configurations from the module."""
+    return local_internal_options
+
+
+@pytest.fixture(scope="module", params=configurations, ids=configuration_ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+def test_options_state_interval_no_file(get_local_internal_options, get_configuration,
+                                        configure_environment, restart_logcollector):

```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over. That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
In case of the need of all Wazuh daemon, the test should disable every module not necessary for the testing in ossec.conf.

The reasons for this change are:

- Faster testing
- More encapsulated tests